scalableminds/webknossos
# Refactor backend: Merge TeamSQL and Team

### URL of deployed dev instance (used for testing):
- https://refactorbackendteam.webknossos.xyz

### Steps to test:
- create and delete teams
- assign and unassign teams to users, also to datasets
- users should see the right datasets
- upload dataset
- create an organization

### Issues:
- contributes to #2521 
- contributes to #2447 

------
